src/SourceGenerators/SourceGeneratorHelpers/Helpers/UnoNamedTypeSymbolDisplayVisitor.cs

```diff
@@ -0,0 +1,144 @@
+#nullable enable
```

Comment:
If there's a source file reference in roslyn that we need to keep track of for future updates, let's add it here.

Reply:
@jeromelaban There is not. This code isn't taken from anywhere :)